docs/content/tags.yml

```
@@ -251,3 +251,7 @@ setup:
 testing:
   label: Testing
   description: Process of evaluating code.
+
+iota-wallet:
+  label: IOTA Wallet
+  description: IOTA Wallet Chrome Extension
```

I wouldnt call it Chrome Extension because it can be used in any Chromium based explorer too like Brave, Edge, ... I would better use: 
```suggestion
  description: IOTA Wallet Browser Extension
```